spec/views/layouts/sidebar.html.erb_spec.rb

```diff
@@ -145,17 +145,4 @@ def true_user
       expect(rendered).to_not have_link("Emancipation Checklist", href: "/emancipation_checklists")
     end
   end
-
-  context "impersonation" do
-    let(:user) { build_stubbed :volunteer }
-    let(:true_user) { build_stubbed :casa_admin }
-
-    it "renders a stop impersonating link when impersonating" do
-      allow(view).to receive(:true_user).and_return(true_user)
-
-      render partial: "layouts/sidebar"
-
-      expect(rendered).to have_link(href: "/volunteers/stop_impersonating")
-    end
```

Comment:
This ought to move to https://github.com/rubyforgood/casa/blob/d7061ea1c9540735038a594422643328e29fb78b/spec/views/layouts/header.html.erb_spec.rb#L112

Reply:
+1, don't delete the test just move it
After that, looks good to merge

Reply:
Ahh good point. Let me handle this tomorrow.  Just got back from a very delayed flight.